src/Orleans.Core/Logging/ILoggerExtensions.cs

```diff
@@ -275,7 +275,9 @@ internal static void Fail(this ILogger logger, ErrorCode errorCode, string messa
             {
                 logger.Error(ErrorCode.Logger_ProcessCrashing, "INTERNAL FAILURE! Process crashing!");
 
-                Environment.FailFast("Unrecoverable failure: " + message);
+                // Environment.FailFast triggers a popup on some machine "xxx has stopped working"
+                // Environment.FailFast("Unrecoverable failure: " + message);
+                Environment.Exit(128);
```

Comment:
Delete the commented code?

Reply:
I wanted to keep it if we want to switch again, but yes I can delete it